cmd/thanos/store.go

```diff
@@ -308,8 +310,11 @@ func runStore(
 	if err != nil {
 		return err
 	}
-
-	bkt, err := client.NewBucket(logger, confContentYaml, conf.component.String())
+	customBktConfig := exthttp.DefaultCustomBucketConfig()
```

Comment:
:grimacing: this actually now always enables hedged requests. Perhaps we need to add `enabled: false/true` flag & check it inside of `CreateHedgedTransportWithConfig`?